packages/dashboard/src/components/sidePanel/sections/thresholdsSection/defaultValues.ts

```diff
@@ -1,4 +1,4 @@
-import { COMPARISON_OPERATOR } from '@synchro-charts/core';
+import { ComparisonOperator } from '@iot-app-kit/core';
 
 export const DEFAULT_THRESHOLD_COLOR = '#D0021B';
-export const OPS_ALLOWED_WITH_STRING: COMPARISON_OPERATOR[] = [COMPARISON_OPERATOR.EQUAL, COMPARISON_OPERATOR.CONTAINS];
+export const OPS_ALLOWED_WITH_STRING: ComparisonOperator[] = ['EQ', 'CONTAINS'];
```

Comment:
we have a COMPARISON_OPERATOR in core which defines these values :)

Reply:
Good call. updating it